tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -221,13 +220,24 @@ func (c *CustomClusterProvisioningTestSuite) testProvisioningK3SCustomCluster(cl
 	err = wait.WatchWait(result, checkFunc)
 	assert.NoError(c.T(), err)
 	assert.Equal(c.T(), clusterName, clusterResp.ObjectMeta.Name)
+	assert.Equal(c.T(), kubeVersion, cluster.Spec.KubernetesVersion)
+
+	getClusterName, err := clusters.GetClusterIDByName(c.client, clusterName)
```

Comment:
Nit, I think the returned variable is just a cluster name, in terms of readability. If get is used it might be misunderstood as function, method, etc.

Reply:
yeah, I couldn't think of a good name at the time since `clusterName` is used already. I'll find a better name as I agree that it looks more like a getter/setter terminology than just a variable name.